misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -575,6 +576,17 @@ public RecordFieldNode transform(RecordFieldNode recordField) {
                 .apply();
     }
 
+    @Override
+    public MemberTypeDescriptorNode transform(MemberTypeDescriptorNode member) {
+        NodeList<AnnotationNode> annotations = formatNodeList(member.annotations(),
+                1, 0, 1, 0);
+        TypeDescriptorNode type = formatNode(member.typeDescriptor(), 0, 0);
```

Comment:
We use the parent node's `trailingWS` and `trailingNL` for the last node

```suggestion
        TypeDescriptorNode type = formatNode(member.typeDescriptor(), env.trailingWS, env.trailingNL);
```